// tdesign-vue-next/esm/pro-components.js
/**
 * tdesign v1.13.0
 * (c) 2025 tdesign
 * @license MIT
 */

import { c as components } from './_chunks/dep-1ad354f9.js';
export { C as Chat, g as ChatAction, e as ChatContent, b as ChatInput, a as ChatItem, h as ChatLoading, f as ChatReasoning, d as ChatSender } from './_chunks/dep-1ad354f9.js';
import './chat/chat.js';
import '@babel/runtime/helpers/defineProperty';
import 'vue';
import 'tdesign-icons-vue-next';
import 'lodash-es';
import './chat/props.js';
import './divider/index.js';
import './divider/divider.js';
import './divider/props.js';
import './hooks/useConfig.js';
import './config-provider/hooks/useConfig.js';
import './config-provider/utils/context.js';
import './_chunks/dep-b296c28c.js';
import '@babel/runtime/helpers/typeof';
import './_chunks/dep-d5ba1b57.js';
import 'dayjs';
import './_chunks/dep-b668a2a6.js';
import './utils/render-tnode.js';
import './utils/withInstall.js';
import 'tdesign-vue-next/esm/common/style/web/components/divider/_index.less';
import './popconfirm/index.js';
import './popconfirm/popconfirm.js';
import '@babel/runtime/helpers/slicedToArray';
import './hooks/useGlobalIcon.js';
import './popup/index.js';
import './popup/popup.js';
import '@popperjs/core';
import '@babel/runtime/helpers/toConsumableArray';
import './hooks/useKeepAnimation.js';
import './_chunks/dep-1614d661.js';
import './hooks/useVModel.js';
import './utils/dom.js';
import './popup/container.js';
import './popup/props.js';
import './hooks/useResizeObserver.js';
import 'tdesign-vue-next/esm/common/style/web/components/popup/_index.less';
import './popconfirm/props.js';
import './dialog/hooks/useAction.js';
import './button/index.js';
import './button/button.js';
import './loading/index.js';
import './loading/directive.js';
import './loading/plugin.js';
import './loading/loading.js';
import './loading/icon/gradient.js';
import './_chunks/dep-4eefc93a.js';
import '@babel/runtime/helpers/objectWithoutProperties';
import './hooks/useTeleport.js';
import './loading/props.js';
import 'tdesign-vue-next/esm/common/style/web/components/loading/_index.less';
import './button/props.js';
import './hooks/useRipple.js';
import './hooks/useDisabled.js';
import 'tdesign-vue-next/esm/common/style/web/components/button/_index.less';
import 'tdesign-vue-next/esm/common/style/web/components/popconfirm/_index.less';
import './chat/chat-item.js';
import './chat/chat-item-props.js';
import './skeleton/index.js';
import './skeleton/skeleton.js';
import './skeleton/props.js';
import 'tdesign-vue-next/esm/common/style/web/components/skeleton/_index.less';
import './chat/chat-content.js';
import './chat/chat-content-props.js';
import './_chunks/dep-1a1a5ddb.js';
import '@babel/runtime/helpers/wrapNativeSuper';
import '@babel/runtime/helpers/possibleConstructorReturn';
import '@babel/runtime/helpers/getPrototypeOf';
import '@babel/runtime/helpers/inherits';
import '@babel/runtime/helpers/classCallCheck';
import '@babel/runtime/helpers/createClass';
import './chat/chat-loading.js';
import './chat/chat-loading-props.js';
import './chat/chat-reasoning.js';
import './chat/chat-reasoning-props.js';
import './collapse/index.js';
import './collapse/collapse.js';
import './collapse/props.js';
import './collapse/collapse-panel.js';
import './collapse/collapse-panel-props.js';
import './common-components/fake-arrow.js';
import './hooks/useCollapseAnimation.js';
import 'tdesign-vue-next/esm/common/style/web/components/collapse/_index.less';
import './chat/chat-input.js';
import './textarea/index.js';
import './textarea/textarea.js';
import './form/consts/index.js';
import './hooks/useReadonly.js';
import './input/hooks/useLengthLimit.js';
import './_chunks/dep-988f7f09.js';
import './textarea/props.js';
import 'tdesign-vue-next/esm/common/style/web/components/textarea/_index.less';
import './chat/chat-input-props.js';
import './chat/chat-action.js';
import './chat/chat-action-props.js';
import './space/index.js';
import './space/space.js';
import './space/props.js';
import './hooks/slot.js';
import 'tdesign-vue-next/esm/common/style/web/components/space/_index.less';
import './tooltip/index.js';
import './tooltip/tooltip.js';
import './tooltip/props.js';
import './tooltip/utils/index.js';
import 'tdesign-vue-next/esm/common/style/web/components/tooltip/_index.less';
import './chat/utils/index.js';
import './message/index.js';
import './message/message.js';
import './message/consts/index.js';
import './message/props.js';
import './message/utils/animation.js';
import 'tdesign-vue-next/esm/common/style/web/components/message/_index.less';
import './message/plugin.js';
import './message/message-list.js';
import './notification/index.js';
import './notification/notification.js';
import './notification/props.js';
import './notification/utils/animate.js';
import './notification/consts/index.js';
import 'tdesign-vue-next/esm/common/style/web/components/notification/_index.less';
import './notification/plugin.js';
import './notification/notification-list.js';
import './dialog/index.js';
import './dialog/dialog.js';
import './dialog/props.js';
import './dialog/hooks/useSameTarget.js';
import './hooks/useDestroyOnClose.js';
import './_chunks/dep-e77876a7.js';
import './hooks/usePopupManager.js';
import './dialog/utils/index.js';
import './dialog/dialog-card.js';
import './dialog/dialog-card-props.js';
import 'tdesign-vue-next/esm/common/style/web/components/dialog/_index.less';
import './dialog/plugin.js';
import './drawer/index.js';
import './drawer/drawer.js';
import './drawer/props.js';
import './drawer/hooks/index.js';
import 'tdesign-vue-next/esm/common/style/web/components/drawer/_index.less';
import './drawer/plugin.js';
import './chat/chat-sender.js';
import './chat/chat-sender-props.js';
import 'tdesign-vue-next/esm/common/style/web/components/chat/_index.less';
import 'tdesign-vue-next/esm/common/style/web/components/chat/atom-one-dark.less';

function install(app, config) {
  Object.keys(components).forEach(function (key) {
    if (/directive/i.test(key)) return;
    /plugin/i.test(key) ? app.use(components[key]) : app.use(components[key], config);
  });
}
var proComponents = {
  install: install,
  version: "1.13.0"
};

export { proComponents as default, install };
