// tdesign-vue-next/cjs/_chunks/dep-6ecacf2d.js
/**
 * tdesign v1.12.0
 * (c) 2025 tdesign
 * @license MIT
 */

'use strict';

var chat_chat = require('../chat/chat.js');
var chat_chatItem = require('../chat/chat-item.js');
var chat_chatInput = require('../chat/chat-input.js');
var chat_chatContent = require('../chat/chat-content.js');
var chat_chatReasoning = require('../chat/chat-reasoning.js');
var chat_chatLoading = require('../chat/chat-loading.js');
var chat_chatAction = require('../chat/chat-action.js');
var chat_chatSender = require('../chat/chat-sender.js');
var utils_withInstall = require('../utils/withInstall.js');

var Chat = utils_withInstall["default"](chat_chat["default"]);
var ChatItem = utils_withInstall["default"](chat_chatItem["default"]);
var ChatInput = utils_withInstall["default"](chat_chatInput["default"]);
var ChatSender = utils_withInstall["default"](chat_chatSender["default"]);
var ChatContent = utils_withInstall["default"](chat_chatContent["default"]);
var ChatReasoning = utils_withInstall["default"](chat_chatReasoning["default"]);
var ChatAction = utils_withInstall["default"](chat_chatAction["default"]);
var ChatLoading = utils_withInstall["default"](chat_chatLoading["default"]);

var components = /*#__PURE__*/Object.freeze({
  __proto__: null,
  Chat: Chat,
  ChatItem: ChatItem,
  ChatInput: ChatInput,
  ChatSender: ChatSender,
  ChatContent: ChatContent,
  ChatReasoning: ChatReasoning,
  ChatAction: ChatAction,
  ChatLoading: ChatLoading,
  'default': Chat
});

exports.Chat = Chat;
exports.ChatAction = ChatAction;
exports.ChatContent = ChatContent;
exports.ChatInput = ChatInput;
exports.ChatItem = ChatItem;
exports.ChatLoading = ChatLoading;
exports.ChatReasoning = ChatReasoning;
exports.ChatSender = ChatSender;
exports.components = components;
