// tdesign-vue-next/es/_chunks/dep-e8a652db.mjs
/**
 * tdesign v1.12.0
 * (c) 2025 tdesign
 * @license MIT
 */

import _Chat from '../chat/chat.mjs';
import _ChatItem from '../chat/chat-item.mjs';
import _ChatInput from '../chat/chat-input.mjs';
import _ChatContent from '../chat/chat-content.mjs';
import _ChatReasoning from '../chat/chat-reasoning.mjs';
import _ChatLoading from '../chat/chat-loading.mjs';
import _ChatAction from '../chat/chat-action.mjs';
import _ChatSender from '../chat/chat-sender.mjs';
import withInstall from '../utils/withInstall.mjs';
import './style/css.mjs';

var Chat = withInstall(_Chat);
var ChatItem = withInstall(_ChatItem);
var ChatInput = withInstall(_ChatInput);
var ChatSender = withInstall(_ChatSender);
var ChatContent = withInstall(_ChatContent);
var ChatReasoning = withInstall(_ChatReasoning);
var ChatAction = withInstall(_ChatAction);
var ChatLoading = withInstall(_ChatLoading);

var components = /*#__PURE__*/Object.freeze({
  __proto__: null,
  Chat: Chat,
  ChatItem: ChatItem,
  ChatInput: ChatInput,
  ChatSender: ChatSender,
  ChatContent: ChatContent,
  ChatReasoning: ChatReasoning,
  ChatAction: ChatAction,
  ChatLoading: ChatLoading,
  'default': Chat
});

export { Chat as C, ChatItem as a, ChatInput as b, components as c, ChatSender as d, ChatContent as e, ChatReasoning as f, ChatAction as g, ChatLoading as h };
