// tdesign-vue-next/esm/_chunks/dep-5c57e73e.js
/**
 * tdesign v1.12.0
 * (c) 2025 tdesign
 * @license MIT
 */

import _Chat from '../chat/chat.js';
import _ChatItem from '../chat/chat-item.js';
import _ChatInput from '../chat/chat-input.js';
import _ChatContent from '../chat/chat-content.js';
import _ChatReasoning from '../chat/chat-reasoning.js';
import _ChatLoading from '../chat/chat-loading.js';
import _ChatAction from '../chat/chat-action.js';
import _ChatSender from '../chat/chat-sender.js';
import withInstall from '../utils/withInstall.js';
import 'tdesign-vue-next/esm/common/style/web/components/chat/_index.less';
import 'tdesign-vue-next/esm/common/style/web/components/chat/atom-one-dark.less';

var Chat = withInstall(_Chat);
var ChatItem = withInstall(_ChatItem);
var ChatInput = withInstall(_ChatInput);
var ChatSender = withInstall(_ChatSender);
var ChatContent = withInstall(_ChatContent);
var ChatReasoning = withInstall(_ChatReasoning);
var ChatAction = withInstall(_ChatAction);
var ChatLoading = withInstall(_ChatLoading);

var components = /*#__PURE__*/Object.freeze({
  __proto__: null,
  Chat: Chat,
  ChatItem: ChatItem,
  ChatInput: ChatInput,
  ChatSender: ChatSender,
  ChatContent: ChatContent,
  ChatReasoning: ChatReasoning,
  ChatAction: ChatAction,
  ChatLoading: ChatLoading,
  'default': Chat
});

export { Chat as C, ChatItem as a, ChatInput as b, components as c, ChatSender as d, ChatContent as e, ChatReasoning as f, ChatAction as g, ChatLoading as h };
