// tdesign-vue-next/es/pro-components.mjs
/**
 * tdesign v1.13.0
 * (c) 2025 tdesign
 * @license MIT
 */

import { c as components } from './_chunks/dep-2fb3a957.mjs';
export { C as Chat, g as ChatAction, e as ChatContent, b as ChatInput, a as ChatItem, h as ChatLoading, f as ChatReasoning, d as ChatSender } from './_chunks/dep-2fb3a957.mjs';
import './common.mjs';
import './chat/chat.mjs';
import './_chunks/dep-4c19ba66.mjs';
import './_chunks/dep-3a46b404.mjs';
import 'vue';
import 'tdesign-icons-vue-next';
import 'lodash-es';
import './chat/props.mjs';
import './divider/index.mjs';
import './divider/divider.mjs';
import './divider/props.mjs';
import './hooks/useConfig.mjs';
import './config-provider/hooks/useConfig.mjs';
import './config-provider/utils/context.mjs';
import './_chunks/dep-960f05e3.mjs';
import './_chunks/dep-608ebfe7.mjs';
import './_chunks/dep-0f539ccd.mjs';
import './_chunks/dep-3873beae.mjs';
import './_chunks/dep-8664c288.mjs';
import './utils/render-tnode.mjs';
import './utils/withInstall.mjs';
import './style/css.mjs';
import './popconfirm/index.mjs';
import './popconfirm/popconfirm.mjs';
import './_chunks/dep-c9dbce85.mjs';
import './_chunks/dep-52e265eb.mjs';
import './hooks/useGlobalIcon.mjs';
import './popup/index.mjs';
import './popup/popup.mjs';
import '@popperjs/core';
import './hooks/index.mjs';
import './hooks/icon.mjs';
import './hooks/slot.mjs';
import './_chunks/dep-c7a7c399.mjs';
import './hooks/useCommonClassName.mjs';
import './hooks/useDefaultValue.mjs';
import './hooks/useDestroyOnClose.mjs';
import './hooks/useKeepAnimation.mjs';
import './hooks/useRipple.mjs';
import './_chunks/dep-487987c7.mjs';
import './hooks/useVirtualScroll.mjs';
import './hooks/useVModel.mjs';
import './hooks/useImagePreviewUrl.mjs';
import './_chunks/dep-9f0c196e.mjs';
import './_chunks/dep-2f8437f0.mjs';
import './utils/dom.mjs';
import './popup/container.mjs';
import './popup/props.mjs';
import './hooks/useResizeObserver.mjs';
import './popconfirm/props.mjs';
import './dialog/hooks/index.mjs';
import './dialog/hooks/useAction.mjs';
import './button/index.mjs';
import './button/button.mjs';
import './loading/index.mjs';
import './loading/directive.mjs';
import './loading/plugin.mjs';
import './loading/loading.mjs';
import './loading/icon/gradient.mjs';
import './_chunks/dep-b638a875.mjs';
import './_chunks/dep-c514ed63.mjs';
import './hooks/useTeleport.mjs';
import './loading/props.mjs';
import './button/props.mjs';
import './hooks/useDisabled.mjs';
import './dialog/hooks/useSameTarget.mjs';
import './chat/chat-item.mjs';
import './chat/chat-item-props.mjs';
import './skeleton/index.mjs';
import './skeleton/skeleton.mjs';
import './skeleton/props.mjs';
import './chat/chat-content.mjs';
import './chat/chat-content-props.mjs';
import './_chunks/dep-43bab2f6.mjs';
import './_chunks/dep-e0ea9e80.mjs';
import './chat/chat-loading.mjs';
import './chat/chat-loading-props.mjs';
import './chat/chat-reasoning.mjs';
import './chat/chat-reasoning-props.mjs';
import './collapse/index.mjs';
import './collapse/collapse.mjs';
import './collapse/props.mjs';
import './collapse/collapse-panel.mjs';
import './collapse/collapse-panel-props.mjs';
import './common-components/fake-arrow.mjs';
import './hooks/useCollapseAnimation.mjs';
import './chat/chat-input.mjs';
import './textarea/index.mjs';
import './textarea/textarea.mjs';
import './form/consts/index.mjs';
import './hooks/useReadonly.mjs';
import './input/hooks/useLengthLimit.mjs';
import './_chunks/dep-9e34a8ce.mjs';
import './textarea/props.mjs';
import './chat/chat-input-props.mjs';
import './chat/chat-action.mjs';
import './chat/chat-action-props.mjs';
import './space/index.mjs';
import './space/space.mjs';
import './space/props.mjs';
import './tooltip/index.mjs';
import './tooltip/tooltip.mjs';
import './tooltip/props.mjs';
import './tooltip/utils/index.mjs';
import './chat/utils/index.mjs';
import './plugins.mjs';
import './message/index.mjs';
import './message/message.mjs';
import './message/consts/index.mjs';
import './message/props.mjs';
import './message/utils/index.mjs';
import './message/utils/animation.mjs';
import './message/plugin.mjs';
import './message/message-list.mjs';
import './notification/index.mjs';
import './notification/notification.mjs';
import './notification/props.mjs';
import './notification/utils/index.mjs';
import './notification/utils/animate.mjs';
import './notification/consts/index.mjs';
import './notification/plugin.mjs';
import './notification/notification-list.mjs';
import './dialog/index.mjs';
import './dialog/dialog.mjs';
import './dialog/props.mjs';
import './_chunks/dep-eaa30704.mjs';
import './hooks/usePopupManager.mjs';
import './dialog/utils/index.mjs';
import './dialog/dialog-card.mjs';
import './dialog/dialog-card-props.mjs';
import './dialog/plugin.mjs';
import './drawer/index.mjs';
import './drawer/drawer.mjs';
import './drawer/props.mjs';
import './drawer/hooks/index.mjs';
import './drawer/plugin.mjs';
import './chat/chat-sender.mjs';
import './chat/chat-sender-props.mjs';

function install(app, config) {
  Object.keys(components).forEach(function (key) {
    if (/directive/i.test(key)) return;
    /plugin/i.test(key) ? app.use(components[key]) : app.use(components[key], config);
  });
}
var proComponents = {
  install: install,
  version: typeof "1.13.0" === "undefined" ? "" : "1.13.0"
};

export { proComponents as default, install };
